milli/src/search/query_tree.rs

```diff
@@ -294,48 +294,6 @@ fn synonyms(ctx: &impl Context, word: &[&str]) -> heed::Result<Option<Vec<Operat
     }))
 }
 
-/// The query tree builder is the interface to build a query tree.
```

Comment:
Could you fix all the warnings in the imports introduced when you moved the `MatchingWords` struct away, please?